yaml/pois.yaml

```diff
@@ -2206,6 +2206,7 @@ filters:
     output:
       <<: *output_properties
       kind: viewpoint
+      direction: {col: direction}
```

Comment:
There are many more POIs that include `direction`, so let's move this to the very top of the YAML doc after     `wikidata_id: {col: wikidata}` in the 
 

```
global:
  - &output_properties:
```

section, please.